### PR TITLE
DP-1489 Port over csv-validator

### PR DESCRIPTION
**Pull request by simenheg**

### CODE REVIEW

File: okdata/pipeline/validators/csv/jsonschema_validator.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    jsonschema_year,
+)
+
+SCHEMA_SUPPORTED_VERSIONS = ["http://json-schema.org/draft-07/schema#"]
+SCHEMA_FORMATTERS = {
+    "http://json-schema.org/draft-07/schema#": jsonschema.draft7_format_checker
+}
+
+
+class JsonSchemaValidator:
```
Comment (fredriv) [inline okdata/pipeline/validators/csv/jsonschema_validator.py:15]: Use the class from json-validator (https://github.com/oslokommune/okdata-pipeline/pull/8) ?

File: okdata/pipeline/validators/csv/jsonschema_checkers.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from dateutil.parser import isoparse
+
+
+def jsonschema_datetime(value):
```
Comment (fredriv) [inline okdata/pipeline/validators/csv/jsonschema_checkers.py:4]: Use the one from json-validator (https://github.com/oslokommune/okdata-pipeline/pull/8) ?

File: test/validators/csv/jsonschema_validator_test.py [hunk 3]
```diff
@@ -0,0 +1,85 @@
+import pytest
+
+from okdata.pipeline.validators.csv.jsonschema_validator import JsonSchemaValidator
+from okdata.pipeline.validators.csv.parser import parse_csv
+
+
+class TestValidJsonSchema:
```
Comment (fredriv) [inline test/validators/csv/jsonschema_validator_test.py:7]: See if there are any tests that should be merged into https://github.com/oslokommune/okdata-pipeline/pull/8 ?

**Review by fredriv — APPROVED** [review]
Should reuse Json validator classes from https://github.com/oslokommune/okdata-pipeline/pull/8 - otherwise LGTM :+1:

**Comment by simenheg** [comment]
> Should reuse Json validator classes from #8 - otherwise LGTM +1

Will do in a separate PR!

**Pull request merged**